### ISSUE TITLE
digest cycle per frame

### ISSUE THREAD

**Issue by geiregjo** [tusharghate/angular-simple-sprite#3]
I've noticed that using simple-sprite directive in my project increased the $digest cycles. I use it to render a spinner/loader. I think that per frame a $digest cycle is triggered. Is this wanted behavior?

**Comment by tusharghate** [comment]
Hey @geiregjo, 

First off, thanks for using the simple-sprite directive. In regards to the issue you're experiencing -- no, this isn't the wanted behaviour. 
1. There is a two way binding on the "frames", "framesWidth", and "framesHeight". I've removed this in the newest build.
2. I am using $interval to animate between the frames, which would trigger a digest cycle after each interval. In most cases this may be the wanted behaviour, but in this case, it's not. I've modified the code to use the default "setInterval", so now the digest cycle should not be triggered after each iteration. 

Fixed in [version 0.0.6](https://github.com/tusharghate/angular-simple-sprite/releases/tag/v0.0.6)

Thanks,
Tushar
